src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -138,6 +138,7 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
                 // commandline in an existing window, or a new one, but either way,
                 // this process doesn't need to make a new window.
 
+                AllowSetForegroundWindow(static_cast<DWORD>(_monarch.GetPID()));

```

Comment:
we could technically 
```suggestion
                LOG_IF_WIN32_BOOL_FALSE(AllowSetForegroundWindow(static_cast<DWORD>(_monarch.GetPID())));
```

But that's not worth blocking over. I don't think that would be too noisy - at most once per window.

Reply:
(i'm 85% sure that's the right `wil` logging macro)